tier2/hooks/post_gen_project.py

```diff
@@ -26,12 +27,54 @@ def addTopic():
     gh_cli_command = [
         "gh", "repo", "edit",
         f"{ORG_NAME}/{REPO_NAME}",
-        "--add-topic=dsacms-tier0",
+        "--add-topic=dsacms-tier2",
     ]
     subprocess.call(gh_cli_command)
 
+def addMaintainer():
+    maintainers = []
+    add_maintainer = True
+    while add_maintainer:
+        maintainer = {}
+        maintainer["role"] = input("Maintainer's Role (Reviewer, Approver, Maintainer): ").strip()
+        maintainer["name"] = input("Maintainer's Name: ").strip()
+        maintainer["github_username"] = input("Maintainer's GitHub Username: ").strip()
+        maintainer["affiliation"] = input("Maintainer's Affiliation (DSAC, CCSQ, CMMI, etc...): ").strip()
+        maintainers.append(maintainer)
+
+        while True:
+            add_maintainer_input = input("Would you like to add another maintainer? [Y/n]: ").strip().lower()
+            if add_maintainer_input in ("y", ""):  # Empty string for just pressing Enter
```

Comment:
This should handle if the user types in 'Y' or 'yes' or 'Yes' as well

Reply:
Oh I forgot about the `.lower()` statement so that aspect is already handled. However, the script should handle if the user types in something like 'yes' or 'no' instead of 'y' or 'n'.

Reply:
@IsaacMilarky Fixed!